File: app/src/main/java/in/ac/bvmengineering/udaan2k19/Misc/Constant.java
package in.ac.bvmengineering.udaan2k19.Misc;

import android.content.Context;
import android.view.animation.Animation;
import android.view.animation.AnimationUtils;

import in.ac.bvmengineering.udaan2k19.R;

public class Constant {
    public final static String AUTOMOTIVE_PHILOSOPHER = "mechProd";
    public final static String BUILDER_OF_AZKABAN = "civil";
    public final static String CHAMBER_OF_CODERS = "cpit";
    public final static String HALF_WAVE_PRINCE = "ecel";
    public final static String ORDER_OF_OHMS = "ee";
    public static final String GOBLET_OF_WORKSHOPS = "workshop";
    public static final String SCAMANDERS_SUITCASE = "non-tech";
    public static final String MAD_HOLLOWS = "cultural";
    public static final String ANDROID_DEVELOPER = "Android Developer";
    public static final String DEVELOPER_HEAD = "Developer Head";
    public static final String WEB_DEVELOPER = "Web Developer";
    public static final String UI_DESIGNER = "UI Designer";
    public static final String[] EVENT_CATEGORIES = {BUILDER_OF_AZKABAN, AUTOMOTIVE_PHILOSOPHER, CHAMBER_OF_CODERS, HALF_WAVE_PRINCE, MAD_HOLLOWS, SCAMANDERS_SUITCASE, ORDER_OF_OHMS, GOBLET_OF_WORKSHOPS};
    public static final String[] DEVELOPERS = {DEVELOPER_HEAD, ANDROID_DEVELOPER, UI_DESIGNER, WEB_DEVELOPER};
    public static Animation getAnimation(Context context) {
        return AnimationUtils.loadAnimation(context, R.anim.button_animation);
    }
}

File: app/src/main/java/in/ac/bvmengineering/udaan2k19/Misc/Constant.kt
package `in`.ac.bvmengineering.udaan2k19.Misc

import `in`.ac.bvmengineering.udaan2k19.R
import android.content.Context
import android.view.animation.Animation
import android.view.animation.AnimationUtils

class Constant {

    companion object {
        const val AUTOMOTIVE_PHILOSOPHER = "mechProd"
        const val BUILDER_OF_AZKABAN = "civil"
        const val CHAMBER_OF_CODERS = "cpit"
        const val HALF_WAVE_PRINCE = "ecel"
        const val ORDER_OF_OHMS = "ee"
        const val GOBLET_OF_WORKSHOPS = "workshop"
        const val SCAMANDERS_SUITCASE = "non-tech"
        const val MAD_HOLLOWS = "cultural"
        const val ANDROID_DEVELOPER = "Android Developer"
        const val DEVELOPER_HEAD = "Developer Head"
        const val WEB_DEVELOPER = "Web Developer"
        const val UI_DESIGNER = "UI Designer"
        val EVENT_CATEGORIES = arrayOf(BUILDER_OF_AZKABAN, AUTOMOTIVE_PHILOSOPHER, CHAMBER_OF_CODERS, HALF_WAVE_PRINCE, MAD_HOLLOWS, SCAMANDERS_SUITCASE, ORDER_OF_OHMS, GOBLET_OF_WORKSHOPS)
        val DEVELOPERS = arrayOf(DEVELOPER_HEAD, ANDROID_DEVELOPER, UI_DESIGNER, WEB_DEVELOPER)
        fun getAnimation(context: Context?): Animation {
            return AnimationUtils.loadAnimation(context, R.anim.button_animation)
        }
    }
}
